src/test/rgw/rgw_multi/tests_ps.py

```diff
@@ -314,19 +315,23 @@ def verify_s3_records_by_elements(records, keys, exact_match=False, deletions=Fa
                         record['s3']['object']['key'] == key.name:
                         if deletions and 'ObjectRemoved' in record['eventName']:
                             key_found = True
+                            object_size = record['s3']['object']['size']
                             break
                         elif not deletions and 'ObjectCreated' in record['eventName']:
                             key_found = True
+                            object_size = record['s3']['object']['size']
                             break
         else:
             for record in records['Records']:
                 if record['s3']['bucket']['name'] == key.bucket.name and \
                     record['s3']['object']['key'] == key.name:
                     if deletions and 'ObjectRemoved' in record['eventName']:
                         key_found = True
+                        object_size = record['s3']['object']['size']
                         break
                     elif not deletions and 'ObjectCreated' in record['eventName']:
-                        key_found = True
+                        pkey_found = True
```

Comment:
this is an introduced bug in the test. see here: 
http://qa-proxy.ceph.com/teuthology/yuvalif-2020-04-22_12:08:46-rgw:multisite-fix_zer_size_in_notifications-distro-basic-smithi/4975219/teuthology.log